src/test/java/org/primeframework/mvc/test/RequestBuilder.java

```diff
@@ -847,6 +847,11 @@ HTTPResponseWrapper run() {
     }
 
     List<Locale> locales = request.getLocales();
+    if (locales.isEmpty()) {
+      // request.getLocale() returns a default locale if none are set by httpRequestConsumer but
+      // we still want to set to the system's default, if none were explicitly set.
+      locales = List.of(Locale.getDefault());
```

Comment:
Can't do it the old way, because now with multiple header support working correctly, that breaks `LocaleActionTest`